pkg/middlewares/aws_route_table_expander_test.go

```diff
@@ -63,11 +63,21 @@ func TestAwsRouteTableExpander_Execute(t *testing.T) {
 			},
 		},
 		{
-			"test route are expanded",
+			"test routes are expanded except when it already exist",
```

Comment:
Can you, instead of changing the two current tests, create a new one. One test that won't expand already created routes in a default and a non-default route-table ?